### ISSUE TITLE
Cover targets from package.json don't work

### ISSUE THREAD

**Issue by yurishkuro** [uber/jaeger-client-node#25]
```
$ npm run report-cover

> jaeger-client@0.1.0 report-cover ...
> nyc report --reporter html --reporter text

sh: nyc: command not found
```

**Comment by yurishkuro** [comment]
the report-cover command doesn't work even after `nvm install nyc`
```
$ npm run report-cover

> jaeger-client@0.1.0 report-cover ...
> nyc report --reporter html --reporter text

fs.js:808
  return binding.readdir(pathModule._makeLong(path));
                 ^

Error: ENOENT: no such file or directory, scandir '/Users/yurishkuro/Uber/jaeger-client-node/.nyc_output'
```

**Comment by yurishkuro** [comment]
After I added `"nyc": "2.4.0",` to package.json, I was able to run `report-cover`, but not `npm run check-cover`

```
$ npm run check-cover

> jaeger-client@0.1.0 check-cover ...
> nyc check-coverage --branches=10 --lines=10 --functions=10

events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: spawn /.../jaeger-client-node/node_modules/nyc/node_modules/.bin/istanbul ENOENT
    at exports._errnoException (util.js:870:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at nextTickCallbackWith2Args (node.js:437:9)
    at process._tickCallback (node.js:351:17)
    at Function.Module.runMain (module.js:443:11)
    at startup (node.js:134:18)
    at node.js:962:3
```

**Comment by yurishkuro** [comment]
Resolved by #28
